src/time.cr

```diff
@@ -252,6 +252,13 @@ struct Time
   # :nodoc:
   DAYS_PER_4_YEARS = 365*4 + 1
 
+  # This constant is defined to be "1970-01-01 00:00:00 UTC" on all systems with respect to the system clock.
+  # Can be used to create a `Time::Span` that represents an unix epoch time duration.
+  # ```
```

Comment:
Missing blank line above.

Reply:
Maybe this should be done by the formatter. WDYT?